src/tasks/wifi.c

```diff
@@ -526,23 +630,30 @@ static void _task(void *params)
                                    portMAX_DELAY))
                         continue;
 
+                if (!esp8266_drv_is_initialized())
```

Comment:
If we do a hard reset while the system is currently running, existing timers will flood the esp8266 with requests (beacon, camera control, etc). This blocks commands from happening until we're up and running.